<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntroduceWidget</class>
 <widget class="QWidget" name="IntroduceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IntroduceWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#IntroduceWidget{
	
	background-color: rgb(255, 255, 255);
}
QWidget#IntroduceWidget QWidget{
	border:2px solid  rgb(221, 221, 221);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="widgetLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="introduceLayout">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="introduceLayout1">
         <property name="spacing">
          <number>20</number>
         </property>
         <item>
          <widget class="QWidget" name="teamWidget1" native="true">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>实验室团队介绍图片/视频</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="teamWidget2" native="true">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>开发团队介绍</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="introduceLayout2">
         <item>
          <layout class="QVBoxLayout" name="productLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="productLabel">
             <property name="styleSheet">
              <string notr="true">QLabel{
	border-color:rgb(221, 221, 221) rgb(221, 221, 221) rgb(76, 76, 76)  rgb(221, 221, 221);
}</string>
             </property>
             <property name="text">
              <string>产品列表</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="product" native="true">
             <property name="styleSheet">
              <string notr="true">QWidget{
	border-width:0px 2px 2px 2px
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="functionLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="functionLabel">
             <property name="styleSheet">
              <string notr="true">QLabel{
	border-color:rgb(221, 221, 221) rgb(221, 221, 221) rgb(76, 76, 76)  rgb(221, 221, 221);
}</string>
             </property>
             <property name="text">
              <string>功能介绍</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="function" native="true">
             <property name="styleSheet">
              <string notr="true">QWidget{
	border-width:0px 2px 2px 2px
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="contactLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="contactLabel">
             <property name="styleSheet">
              <string notr="true">QLabel{
	border-color:rgb(221, 221, 221) rgb(221, 221, 221) rgb(76, 76, 76)  rgb(221, 221, 221);
}</string>
             </property>
             <property name="text">
              <string>联系我们</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="contact" native="true">
             <property name="styleSheet">
              <string notr="true">QWidget{
	border-width:0px 2px 2px 2px
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="textLayout">
       <property name="spacing">
        <number>30</number>
       </property>
       <item>
        <widget class="QWidget" name="textWidget1" native="true">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>文字</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="textWidget2" native="true">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>文字</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="textWidget3" native="true">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>文字</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
